shared/schemas/workflow-option-schema.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "WorkflowOption",
+    "description": "Workflow result option, key:value pairs.",
+    "type": "object",
+    "properties": {
+        "key": {
+            "description": "Unique identifier of the option. Preferably human-readable.",
+            "type": "string"
+        },
+        "value": {
+            "description": "Free form value of the option.",
+            "type": "string"
+        },
+        "valueType": {
```

Comment:
Or we can provide a property per each type.
Instead of a single `value`, we can have `number`, `text`, `link`. I prefer single `value` and having sort of the `valueType` to guide next processing (e.g. UI decisions)

Reply:
This suggestion looks good to me [key,value,valueType].
Should there be a default value for the `valueType` (e.g. string) ?
@batzionb?

Reply:
I am in favor of having well-defined types and not just arbitrary key-value pairs. if you do go with that option then remove the valueType. Always a string and it is up to the code that uses the key-value to interpret the string.

Reply:
The UI will not know to to treat a `string` value that represents a number or URL.

Reply:
why not? sure it will. based on the name. E.g. key=='my-url', value=='http://cnn.com', key=='my-age', value=='49'. and if we say this is not good then we need to define an exact object

Reply:
Per offline discussion, we are introducing new optional `format` property to avoid such heuristics. It will add us more options for extension as well.